docs/tutorials/noise.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<p>concetrating -&gt; concentrating</p><p>Doing ... are -&gt; Doing ... is</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/quantum/pull/582/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>